### PR TITLE
fixing failed tests

### PR DESCRIPTION
**Pull request by Asai-Yume**

### CODE REVIEW

File: stepcvt/project.py [hunk 1]
```diff
@@ -86,9 +86,9 @@ def load_step_file(cls, name: str, path: Path):
         # the loaded file can be a hidden attribute on CADSource
         cs = cls(name=name, path=path)
         sr = stepreader.StepReader()
-
-        cs._CADSource__step = sr.load(str(path))
-
+        sr.load(str(path))
+        cs._CADSource__step = sr
+        print("changed version")
```
Comment (sree314) [inline stepcvt/project.py:91]: Can you remove this debug print?

**Comment by Asai-Yume** [comment]
I looked at the book step file made in models.py. The part_id should be "book_body", but the test is failing because the method appended a "_part" to "book_body", should the test be changed? Or is it something in model.py that I am missing?

**Comment by Asai-Yume** [comment]
> I looked at the book step file made in models.py. The part_id should be "book_body", but the test is failing because the method appended a "_part" to "book_body", should the test be changed? Or is it something in model.py that I am missing?

If I change the test into expected_partids = set(["book_body_part", "spine_part"]), the test will pass, but I am not sure why the _part string is being appended to the partids.

**Comment by sree314** [comment]
I have to say I don't know either. Worth discussing tomorrow.

**Comment by Asai-Yume** [comment]
I think this can be merged for now, and I will have to ask Roy to fix whatever issue PartInfo is having.

**Review by sree314 — APPROVED** [review]

**Pull request merged**